src/main/java/com/flightstats/hub/spoke/FileSpokeStore.java

```diff
@@ -178,14 +181,19 @@ private void keysInBucket(String key, OutputStream output) {
         }
         try {
             Collection<File> files;
-            if (resolution.equals("second")) {
+            if (Optional.ofNullable(resolution).orElse("FALSE").equals("second")) {
```

Comment:
Dumb question, but do we have a constant for "second" that would indicate what its context/meaning is? (My brain needs caffeine and I thought "second" meant the thing after "first" and not -- as one should assume with hub things -- a subdivision of a minute. You can probably ignore this comment.)